src/actions/cart.ts

```diff
@@ -2,15 +2,20 @@ import { Product } from '../types/product';
 import { CartTypes } from '../types/cart';
 import { createAction } from '../types/helpers';
 
-export type CartActionTypes = ReturnType<typeof addToCart>
+export type CartActionTypes = ReturnType<typeof addToCart> | ReturnType<typeof removeFromCart>;
 
 // TODO: update cart in local storage with a cart service
 
 export const addToCart = (product: Product)  => {
-  // NOTE: this will be removed in the next PR
-  console.log(`product added: ${JSON.stringify(product)}`);
   return createAction({
     type: CartTypes.AddToCart,
     product,
   });
 };
+
+export const removeFromCart = (location: number) => {
```

Comment:
Minor nit - I got a bit confused by `location` instead of a variable name like `index`. Though if we do end up adding IDs, the point might be moot!

Reply:
Will fix